content/influxdb/v2.0/reference/cli/influxd/upgrade.md

```diff
@@ -9,26 +9,50 @@ products: [oss]
-|:-----|:-----------------|:----------------------------------------------------------------------------------------------|:----------:|
-| `-h` | `--help`         | Help for the `upgrade` command                                                                |            |
-|      | `--config`       | (Optional) Custom InfluxDB path to 1.x config file to upgrade (default: `~/.influxdbv2/conf`) | string     |
-|      | `--v1-meta-dir`  | Path to 1.x meta.db directory (default: `~/.influxdb/meta`)                                   | string     |
-|      | `--v2-bolt-path` | Path to 2.0 BoltDB database (default: `~/.influxdbv2/influxd.bolt`)                           | string     |
+| Flag |                         | Description                                                                                                | Input type |
+|:-----|:------------------------|:-----------------------------------------------------------------------------------------------------------|:----------:|
+| `-m` | `--bolt-path`           | Path for boltdb database (default `~/.influxdbv2/influxd.bolt`)                                            | string     |
+| `-b` | `--bucket`              | Primary bucket name                                                                                        | string     |
+|      | `--config-file`         | (Optional) Custom InfluxDB 1.x config file path, else the default config file                              | string     |
+| `-e` | `--engine-path`         | Path for persistent engine files (default `~/.influxdbv2/engine`)                                          | string     |
+| `-f` | `--force`               | Skip the confirmation prompt                                                                               |            |
+| `-h` | `--help`                | Help for `influxd upgrade`                                                                                 |            |
+|      | `--influx-command-path` | Path to influx command (default `~/go/bin/influx`)                                                         | string     |
+|      | `--log-path`            | (Optional) Custom log file path (default `~/upgrade.log`)                                                  | string     |
+| `-o` | `--org`                 | Primary organization name                                                                                  | string     |
+| `-p` | `--password`            | Password for username                                                                                      | string     |
+| `-r` | `--retention`           | (Optional) Duration bucket will retain data (default `0`; retains data infinitely)                         | string     |
+|      | `--security-script`     | (Optional) Generated security upgrade script path (default `~/influxd-upgrade-security.sh`)                | string     |
```

Comment:
This switch is going away when https://github.com/influxdata/influxdb/pull/19854 is completed and merged